helmcharts/values.yaml

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+k8s:
```

Comment:
Maybe we should also provide namespace with value 'eclipse-che' by default ?

Reply:
Namespace is handled by helmcharts by default:
`helm install --namespace <NAMESPACE> ... `